.github/workflows/ci-cd.yml

```diff
@@ -40,13 +78,13 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ci_node_total: [6]
-        ci_node_index: [0, 1, 2, 3, 4, 5]
+        ci_node_total: [8]
+        ci_node_index: [0, 1, 2, 3, 4, 5, 6, 7]
 
     steps:
       - uses: actions/checkout@v3
       - name: Cache node modules
-        uses: actions/cache@v3
+        uses: actions/cache/restore@v3
```

Comment:
using only the `restore` portion of actions/cache saves us 30 seconds. actions/cache's has a cleanup step that takes up a lot of time. Because we have dedicated `yarn-install` step dedicated before hand, we only need to `restore` the cache here and no worry about packaging up this new cache.